test/snapshots/bundleupdateinteractivetest/test_it_renders_updateable_gems__1.snap.yaml

```diff
@@ -1,37 +1,37 @@
 --- "\e[2;4mname\e[0m                \e[2;4mfrom\e[0m        \e[2;4mto\e[0m       \e[2;4mgroup\e[0m
   \               \e[2;4murl\e[0m\n\e[32maddressable\e[0m         2.8.6    →  2.8.\e[32m7\e[0m
-  \                        \e[34mhttp://example/addressable/v2.8.7\e[0m\n\e[32mbigdecimal\e[0m
```

Comment:
🗒️ This snapshot needed to be updated because the changelog URLs are no longer fake values.